.storybook/config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import 'prismjs';
+import { configure, setAddon } from '@storybook/react';
+import infoAddon from '@storybook/addon-info';
+import { setOptions } from '@storybook/addon-options';
+
+/**
+ * Internal dependencies
+ */
+import * as element from 'element';
+import './style.scss';
+
+function loadStories() {
+	window.wp = { ...window.wp, element };
```

Comment:
Actually, we only need it for `element` because of the pragma defined in the .babelrc.
Hopefully we'd be able to remove `element` and use `React` directly :)